rotkehlchen/chain/evm/constants.py

```diff
@@ -13,6 +13,7 @@
 ZERO_32_BYTES_HEX: Final = '0x' + '0' * 64
 GENESIS_HASH: Final = deserialize_evm_tx_hash(ZERO_32_BYTES_HEX)  # hash for transactions in genesis block # noqa: E501
 EVM_ADDRESS_REGEX: Final = re.compile(r'\b0x[a-fA-F0-9]{40}\b')
+SPAM_TXNS: Final = 'SPAM_TXNS'
```

Comment:
add a comment stating what this constant is. I believe that the abbrevation we use in the code always is

```suggestion
LAST_SPAM_TSX_CACHE: Final = 'SPAM_TXS'
```